src/util.jl

```diff
@@ -53,6 +53,72 @@ function audit(
     end
 end
 
+function update_namescount(
```

Comment:
I would rename this function such that is clearly used for `doc` function. Also, should it be `names_count` (two words)?